src/main/java/uk/gov/hmcts/reform/prl/services/hearings/HearingService.java

```diff
@@ -64,6 +64,15 @@ public Hearings getHearings(String userToken, String caseReferenceNumber) {
                     eachHearing.setUrgentFlag(getUrgentFlagWithInHearing(eachHearing));
                     eachHearing.setHearingTypeValue(getHearingTypeValueWithInHearing(eachHearing,refDataCategoryValueMap));
                 }
+                List<CaseHearing> sortedByLatest = hearings.getCaseHearings().stream().sorted((a,b) -> {
+                    if (a.getNextHearingDate() != null && b.getNextHearingDate() != null) {
```

Comment:
@hmcts-shajahan Do you think we can use Comparator here for sorting ?
e.g `.sorted(Comparator.comparing(m -> m.getValue().getUpdatedTime(), Comparator.reverseOrder()))`